README.md

```diff
@@ -45,6 +45,8 @@ or download the version appropriate for your OS from the [release page](https://
 unzip it,
 and place `elm-format` or `elm-format.exe` (windows) on your `PATH`.
 
+On FreeBSD you can `pkg install hs-elm-format` or use [devel/elm-format](https://www.freshports.org/devel/elm-format/), then execute `elm-format` normally.
```

Comment:
Do you know why the package is called `hs-elm-format` instead of `elm-format` ?  I'd like to add some text about other package managers as well, and it would be great if there could be a consistent package name across systems.

Reply:
I see your point, but it's a FreeBSD thing, basically everything haskell has a hs- prefix.
Same way every python or php package have a python or php prefix. I hope that's not an issue.
Btw 0.8.2 is already installable from FreeBSDs precompiled port packages.

Reply:
ah, yeah, I think that's a bit confusing since elm-format is a command line tool that happens to be written in haskell and not a haskell library.  But thanks for the info.

Reply:
I agree :-D one gets used to it in the OS though.
Thank you for merging and for elm-format!

Reply:
I agree :-D one gets used to it in the OS though.
Thank you for merging and for elm-format!

Reply:
I agree :-D one gets used to it in the OS though.
Thank you for merging and for elm-format!